### DISCUSSION TITLE
feature: Inherit Neighboring Map Properties

### DISCUSSION THREAD

**Discussion by Daywalkrr** [AscensionGameDev/Intersect-Engine#2411] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is a pain to create connecting maps that should share the same properties as one another, especially when maps get complex with custom player lights, fog, sounds, and weather.

**Describe the solution you'd like**
In the map editor, a small button, like a compass, that allows you to click on a cardinal direction in which to inherit the properties of that neighboring map. A confirmation dialog should appear and, when accepted, the current map properties (besides map name) should inherit that of the selected neighbor.

**Comment by jcsnider** [comment]
I was planning on going a completely different direction with this.

I think it would be more beneficial to have an editor for map property presets, and then for each map you can choose a preset or custom properties. 

The reason is updating over time, even if we inherit properties from the start, if we wanna change something small like a soundtrack you are still in purgatory of modifying each individual map.

**Comment by NikoCummings** [comment]
This suggestion is amazing, was going to add the same thing. Creating presets for "zones" that we can load in from a list in the map properties editor.
